test/magpiego/bindings/rabbitmq.go

```diff
@@ -49,6 +50,7 @@ func RabbitMQBinding(envParams map[string]string) BindingStatus {
 		return BindingStatus{false, "Failed to declare a queue"}
 	}
 	msg := "Hello World!"
+	ctx := context.Background()
 	err = ch.PublishWithContext(
 		ctx,
```

Comment:
This is weird that ctx wasn't defined before.. how was it compiling?

Reply:
Super wierd! Is this a global variable defined somewhere? If that's the case we should clean it up.

Reply:
https://github.com/radius-project/radius/blob/main/test/magpiego/bindings/mongo.go#L12

Reply:
We should fix that. Doesn't have to be this PR.

Storing a context in a global variable like that is going to lead to a lot more confusion.